src/main/java/no/difi/sdp/client2/domain/kvittering/Kvitteringsinfo.java

```diff
@@ -0,0 +1,74 @@
+package no.difi.sdp.client2.domain.kvittering;
+
+import java.time.Instant;
+
+public class Kvitteringsinfo {
```

Comment:
Det er felter som ikke trengs for å bekrefte en kvittering, men som _kan_ brukes av en avsender om de vil vite mer om kvitteringen. Jeg er absolutt åpen for forslag her. Ville bare unngå å sende inn 5-6 parametere i konstruktør, så pakket inn feltene.